src/components/residentPermit/VehicleInfo.module.scss

```diff
@@ -27,3 +27,8 @@
     }
   }
 }
+
+.vehicleCopyright {
```

Comment:
Would it be possible to combine some of the common CSS for permitDetail vehicleInfo info into a module?